.github/workflows/e2e.create-docker_based-predicate.schedule.yml

```diff
@@ -29,6 +29,12 @@ jobs:
       - name: Detect the builder ref
         id: detect
         uses: slsa-framework/slsa-github-generator/.github/actions/detect-workflow@main
+      - name: Update the build definition
+        # We use a build definition hard-coded in testadata. To ensure validation against
+        # workflow context, we must update the source references.
+        run: |
+          bdTemplate="internal/builders/docker/testdata/build-definition.json"
+          jq  -r --arg GITHUB_SHA "$GITHUB_SHA" '.externalParameters.source.digest.sha1 = $GITHUB_SHA' "${bdTemplate}" > build-definition.json
```

Comment:
are you sure this will work? The second occurrence of `$GITHUB_SHA` is using single quotes and won't be resolved by the shell

Reply:
sorry, I think that's what `--arg GITHUB_SHA` is for. Ignore my comment

Reply:
yes, exactly! i used this command locally